tests/udp_local_address_test.toit

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2021 Toitware ApS. All rights reserved.
```

Comment:
Just call this `interface_address_test.toit`

Reply:
done.